src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,27 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body,
+div,
+a,
+img,
+ul,
+li,
+header {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
```

Comment:
reset borders, indentation, etc, only for tags you need, f.e: you reset border for the body, but it hasn't default border, so this reseting is redundant

Reply:
ok